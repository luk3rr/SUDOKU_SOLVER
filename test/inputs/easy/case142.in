009000040 078051902 400207500 060000179 002000050 000908004 700089305 080012000 000003006 
