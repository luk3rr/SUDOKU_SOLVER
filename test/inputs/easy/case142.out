Solving the following grid:
0 0 9 | 0 0 0 | 0 4 0 
0 7 8 | 0 5 1 | 9 0 2 
4 0 0 | 2 0 7 | 5 0 0 
------+-------+------
0 6 0 | 0 0 0 | 1 7 9 
0 0 2 | 0 0 0 | 0 5 0 
0 0 0 | 9 0 8 | 0 0 4 
------+-------+------
7 0 0 | 0 8 9 | 3 0 5 
0 8 0 | 0 1 2 | 0 0 0 
0 0 0 | 0 0 3 | 0 0 6 


Solution found :')

5 2 9 | 8 3 6 | 7 4 1 
3 7 8 | 4 5 1 | 9 6 2 
4 1 6 | 2 9 7 | 5 8 3 
------+-------+------
8 6 4 | 3 2 5 | 1 7 9 
9 3 2 | 1 7 4 | 6 5 8 
1 5 7 | 9 6 8 | 2 3 4 
------+-------+------
7 4 1 | 6 8 9 | 3 2 5 
6 8 3 | 5 1 2 | 4 9 7 
2 9 5 | 7 4 3 | 8 1 6 

Algorithm: BFS
Total time: 10 ms
Total expanded states: 1344
